.travis.yml

```diff
@@ -15,8 +15,6 @@ script:
 after_success:
   - coveralls
 env:
-  - DJANGO="Django<1.6"
-  - DJANGO="Django<1.7"
   - DJANGO="Django<1.8"
```

Comment:
This resolves to Django1.7 and can be removed

Reply:
Thanks!